files/ubuntu-16.04/openmpi-modulefile-via-alternatives

```diff
@@ -0,0 +1,3 @@
+#%Module
```

Comment:
can you add more comment (even in the commit log is ok) on how this works?
Would the `update-alternatives` command. be called every time the module is loaded?

Reply:
Yes. I'll add a comment to each of the files. Let me know if you think it needs more information.